pytradfri/device/air_purifier.py

```diff
@@ -15,15 +17,32 @@
     ATTR_AIR_PURIFIER_MODE,
     ATTR_AIR_PURIFIER_MODE_AUTO,
     ATTR_AIR_PURIFIER_MOTOR_RUNTIME_TOTAL,
-    ROOT_AIR_PURIFIER,
+    ATTR_ID,
 )
-from ..typing import AirPurifierResponse
 
 if TYPE_CHECKING:
     # avoid cyclic import at runtime.
     from . import Device
 
 
+class AirPurifierResponse(BaseModel):
```

Comment:
Don't add any new models besides the DeviceResponse model now. The aim should just be to update the device module and keep other changes to a minimum.